src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 1;
+
+  return (num1) => {
+    return (num2) => {
+      if (count < 4 || count % 2 !== 0) {
+        count++;
+
+        return num1 + num2;
+      }
+
+      count++;
+
+      return 'Bzzz... Error!';
+    };
```

Comment:
Pieces of repeatable logic can be moved to a place where they will be called regardless of conditions

```suggestion
    return (num2) => {
      count++;

      if (count < 4 || count % 2 !== 0) {
        return num1 + num2;
      }
 
      return 'Bzzz... Error!';
    };
```